plugins/10_Dragonflight/zones/thaldraszus.lua

```diff
@@ -618,3 +618,67 @@ map.nodes[34676541] = HemetNesingwaryJr({
 map.nodes[50674562] = HemetNesingwaryJr({
     rewards = {Achievement({id = 16542, criteria = 55702})}
 }) -- Northern Thaldraszus Hunt
+
+-------------------------------------------------------------------------------
+-------------------------- ONE OF EVERYTHING, PLEASE --------------------------
+-------------------------------------------------------------------------------
+
+val.nodes[60921096] = Collectible({
+    label = '{item:200904}',
+    icon = 237358,
+    note = L['picante_pomfruit_cake_note'],
+    group = ns.groups.SPECIALTIES,
+    rewards = {Achievement({id = 16621, criteria = 55733})}
+}) -- Picante Pompfruit Cake
```

Comment:
When you set nodes inside the `val` city map, remember to also set `parent = map.id` so that it will appear on the outer zone map as well.

Reply:
Ah, this is my first time needing to do that. I didn't know that was a feature!